script/main.py

```diff
@@ -129,22 +133,28 @@ def create_users_table(conn):
 
 
 def insert_users_to_table(conn, users):
-  sql = '''INSERT INTO users VALUES (?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?)'''
+  sql = '''INSERT INTO users VALUES (?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?)'''
 
   cur = conn.cursor()
   for columns in users:
     cur.execute(sql, columns)  
   conn.commit()
 
 
-def init_db(conn):  
+def init_db(conn, people_json):  
   '''Initializing the database'''    
 
   # create table
   if conn is not None:
     create_users_table(conn)
   else:
-    print("Error! cannot create the database connection.")
+    print("Error! cannot create the database connection.")  
+  
+  (phone_records, dob_records) = find_records(people_json)
```

Comment:
Here before we will fix it.
Could write in plain English steps which are necessary to initialize database?
don't look at those functions yet, since they are not optimal.

Reply:
1. connect to json or api file
2. delete and change some records
3. create a connection with db
4. create a table
5. import of records to db

Was it about something like this?

Reply:
Yes something like that. But those step are not correct. Eg. 
Do you really connect to json API?, 
Do you create connection here or get it as a parameter?
etc.

I think that this ability to write the correct step is something very important which you should learn.
When you have steps writing code is easy :-)
So lets try to improve it!

Reply:
1. take JSON file as parameter
2. add a new record to the JSON file
3. delete and change some records in the JSON file
4. create a connection with db
5. create a table
6. import of records to db

Reply:
Nope :-)

* You don't need to do anything to take parameter
* You need to be more specific: What kind of record you want to add?
* do you really create connection here, do you create table

Write it is such way, that other person will understand what should be done.

Reply:
1. Use a JSON file as a parameter
2. Find the record in the JSON file
3. Add a new record to the JSON file that counts  down to birthday
4. Remove the photo from the records in the JSON file
5. Remove special characters from phone in JSON file
6. Create a table
7. Import of records from a JSON file

Reply:
1. Load a JSON file.
2. Convert the dictionary into a list of records and access individual fields.
3. Add an additional field to the records that calculate how many days are left until the next birthday.
4. Remove of the "photo" field from the records.
5. Remove of special characters from phone and cell fields in all records.
6. Create a table.
7. Insert data into the database.

Reply:
Looks ok to me. The only questions left is:
What is: `access individual fields` in point 2?

As a next step try to rename functions and variables, so the function init() will looks like implementation of this points

Reply:
is access to dob field, phone and cell

Reply:
The changes have been made